design-system/uilib/src/themes/desktop/theme-desktop.svelte

```diff
@@ -54,6 +53,9 @@
 
         font-family: var(--font-regular);
         font-size:   var(--fonr-size);
+
+        --checkbox-animation-length:    700ms;
+        --checkbox-ripple-length:       0.3s;
```

Comment:
These should not be in the theme directly.
We can have the default values in the component and provide the css variables there